MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -26,11 +27,17 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 	func bind() {
 		title = viewModel?.title
 		viewModel?.onLoadingStateChange = { [weak self] isLoading in
-			if isLoading {
-				self?.refreshControl?.beginRefreshing()
-			} else {
-				self?.refreshControl?.endRefreshing()
+			guard let self = self else { return }
+			isLoading ? self.refreshControl?.beginRefreshing() : self.refreshControl?.endRefreshing()
+		}
+
+		viewModel?.onFeedLoadError = { [weak self] error in
+			guard let self = self else { return }
+			guard let errorMessage = error else {
+				self.errorView.hideMessage()
+				return
 			}
+			self.errorView.show(message: errorMessage)
```

Comment:
Suggestion: An `if/else` statement is usually easier to read/follow in such cases:

"If there's a message to show, show it. Else, hide the existing one."

```suggestion
			if let errorMessage = error {
				errorView.show(message: errorMessage)
			} else {
				errorView.hideMessage()
			}
```

Instead of "guard against not having a message. Else, hide the existing one. Show the message if it passed the guard statement."

Reply:
done